### PR TITLE
#MM-34437 added API doc

### PR DESCRIPTION
**Pull request by harshilsharma63**
#### Summary

Added API doc for fetching last trial license

#### Ticket Link
https://mattermost.atlassian.net/browse/MM-34437

### CODE REVIEW

File: v4/source/system.yaml [hunk 1]
```diff
@@ -810,6 +810,28 @@
...
             resp := Client.RequestTrialLicense()
+  /trial-license/prev:
+    get:
+      tags:
+        - system
+      summary: Get last trial license used
+      description: >
+        Get the last trial license used on the sevrer
+
+        __Minimum server version__: 5.35
```
Comment (sbishel) [inline v4/source/system.yaml:821]: This will probably not go in until 5.36.

**Review by sbishel — CHANGES_REQUESTED** [review]

File: v4/source/system.yaml [hunk 2]
```diff
@@ -810,6 +810,28 @@
...
+
+        ##### Permissions
+
+        Must have `manage_systems` permissions.
+      responses:
+        "200":
+          description: Licnse fetched successfully.
```
[the remark this answers is not in the document]
Comment (harshilsharma63) [reply v4/source/system.yaml:828]: Fixed.

File: v4/source/system.yaml (hunk 1 above)
Comment (harshilsharma63) [reply v4/source/system.yaml:821]: Updated the version.

**Comment by harshilsharma63** [comment]
Closing this PR in favour of https://github.com/mattermost/mattermost-api-reference/pull/634 to allow server build to pass.

**Pull request closed without merging**